srcpkgs/linux5.13-surface/files/mv-debug

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Have you considered symlinking `files/` rather than copying?

Reply:
Done